kratos/tests/test_sparse_matrices.py

```diff
@@ -0,0 +1,114 @@
+import KratosMultiphysics
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.kratos_utilities as kratos_utils
+import numpy as np 
```

Comment:
```suggestion

import numpy as np 
```

Separate Kratos imports of other imports